src/lib.rs

```diff
@@ -433,39 +429,16 @@ where
     Router<R>: Send,
     G: GetOriginalDst + Send + 'static,
 {
-    let stack = Arc::new(NewServiceFn::new(move || {
-        // Clone the router handle
-        let router = router.clone();
-
-        // Map errors to appropriate response error codes.
-        let map_err = MapErr::new(router, |e| {
-            match e {
-                RouteError::Route(r) => {
-                    error!(" turning route error: {} into 500", r);
-                    http::StatusCode::INTERNAL_SERVER_ERROR
-                }
-                RouteError::Inner(i) => {
-                    error!("turning {} into 500", i);
-                    http::StatusCode::INTERNAL_SERVER_ERROR
-                }
-                RouteError::NotRecognized => {
-                    error!("turning route not recognized error into 500");
-                    http::StatusCode::INTERNAL_SERVER_ERROR
-                }
-                RouteError::NoCapacity(capacity) => {
-                    // TODO For H2 streams, we should probably signal a protocol-level
-                    // capacity change.
-                    error!("router at capacity ({}); returning a 503", capacity);
-                    http::StatusCode::SERVICE_UNAVAILABLE
-                }
-            }
-        });
 
-        // Install the request open timestamp module at the very top
-        // of the stack, in order to take the timestamp as close as
-        // possible to the beginning of the request's lifetime.
-        telemetry::http::service::TimestampRequestOpen::new(map_err)
-    }));
+
+    // Install the request open timestamp module at the very top of the
```

Comment:
Nit: Since we're using "layer" for what Finagle calls "module", maybe s/module/layer in the comments?